vault/vault-hold-migration-api/src/main/java/com/google/vault/chatmigration/DuplicateHold.java

```diff
@@ -0,0 +1,125 @@
+package com.google.vault.chatmigration;
+
+import com.github.rholder.retry.RetryException;
+import com.google.api.services.vault.v1.Vault;
+import com.google.api.services.vault.v1.model.CorpusQuery;
+import com.google.api.services.vault.v1.model.HeldAccount;
+import com.google.api.services.vault.v1.model.HeldHangoutsChatQuery;
+import com.google.api.services.vault.v1.model.HeldOrgUnit;
+import com.google.api.services.vault.v1.model.Hold;
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.io.StringWriter;
+import java.util.Arrays;
+import java.util.List;
+import java.util.concurrent.ExecutionException;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import java.util.stream.Collectors;
+import org.apache.commons.csv.CSVParser;
+import org.apache.commons.csv.CSVPrinter;
+import org.apache.commons.csv.CSVRecord;
+
+public class DuplicateHold {
+  private static Logger logger = Logger.getLogger(DuplicateHold.class.getName());
+  private static final String ERROR_DESCRIPTION = "Error Description";
+  public static final String HOLD_NAME_SUFFIX = "_hangoutsChat";
+  private static final int MAX_ACCOUNTS_FOR_HOLD = 100;
+
+  private CSVParser csvParser;
+  private boolean includeRooms;
+  private CSVPrinter errorReport;
+  Vault vaultService;
+  private int numberOfHolds = 0;
+
+  public DuplicateHold(
+      CSVParser csvParser, boolean includeRooms, Vault vaultService, CSVPrinter errorReport) {
+    this.csvParser = csvParser;
+    this.errorReport = errorReport;
+    this.includeRooms = includeRooms;
+    this.vaultService = vaultService;
+  }
+
+  public int duplicateHolds() throws Exception {
+    writeHeader();
+    for (CSVRecord record : csvParser.getRecords()) {
+      String matterId = record.get(HoldsReport.MATTER_ID);
+      String name = record.get(HoldsReport.HOLD_NAME);
+      String orgUnitId = record.get(HoldsReport.ORG_UNIT_ID);
+      String accounts = record.get(HoldsReport.ACCOUNT_IDS);
+
+      List<HeldAccount> accountList = (accounts.equals("")) ? null :Arrays.stream(accounts.split(","))
+          .map(account -> new HeldAccount().setAccountId(account))
+          .collect(Collectors.toList());
+      boolean exceedsAccountLimit = false;
+
+      Hold hold = new Hold().setName(name + HOLD_NAME_SUFFIX).setCorpus("HANGOUTS_CHAT");
+
+      if (!"".equals(orgUnitId)) {
```

Comment:
This is a bit backwards.
We should test if `orgUnitId` equals `""`.

Reply:
Got it thanks!